src/io/flutter/logging/FlutterLogView.java

```diff
@@ -39,8 +37,32 @@
 import java.awt.datatransfer.StringSelection;
 import java.awt.event.MouseEvent;
 
+import static com.intellij.ui.SimpleTextAttributes.STYLE_PLAIN;
```

Comment:
We later use other values of SimpleTextAttributes.
Should we statically import all of them, or none?

Reply:
Good catch.  Thanks!